presentation/src/main/java/me/androidbox/presentation/navigation/Screen.kt

```diff
@@ -0,0 +1,13 @@
+package me.androidbox.presentation.navigation
+
+sealed interface Screen {
+    val route: String
+}
```

Comment:
This could be a sealed class Screen(val route: String)

And then you could just

object LoginScreen : NavigationRoute("login_screen")
object RegisterScreen : NavigationRoute("register_screen")